src/mmw/js/src/core/views.js

```diff
@@ -238,9 +240,72 @@ function getLatLngs(boundsOrShape) {
     throw 'Unable to extract latlngs from boundsOrShape argument';
 }
 
+var DeleteModal = Marionette.ItemView.extend({
+    el: '#modal-basic-target',
+
+    viewOptions: ['objToDelete'],
+
+    initialize: function(options) {
+        var self = this;
+        this.mergeOptions(options, this.viewOptions);
+        this.$el.on('hide.bs.modal', function(e) {
+            self.cleanup();
+        });
+    },
+
+    template: modalDeleteTmpl,
+
+    ui: {
+        deleteConfirmed: '.delete',
+    },
+
+    events: {
+        'click @ui.deleteConfirmed': 'deleteModel',
+    },
+
+    deleteModel: function() {
+        this.objToDelete.destroy();
```

Comment:
Seems like we may want the delete modal to just confirm the delete and let the calling controller to the actual deleting.  At the very least, I would want to the jqXHR from this `destroy` returned so the caller could handle success and failure situations.